astroid/interpreter/_import/spec.py

```diff
@@ -378,9 +378,7 @@ def _find_spec_with_path(
 ) -> tuple[Finder | _MetaPathFinder, ModuleSpec]:
     for finder in _SPEC_FINDERS:
         finder_instance = finder(search_path)
```

Comment:
Micro optimization, but seems like we can now move finder_instance below the continue.

Reply:
Ah, no need then. It didn't occur to me there would be a side effect. Fine as is.